flight/mcu/lib/teensyShared/FSMCollection.h

```diff
@@ -1,19 +1,29 @@
 #pragma once
-#include <ChRt.h>
 
-#include "rocketFSM.h"
+#include "ChRt.h"
+#include <initializer_list>
```

Comment:
Comment describing why we need this

Reply:
That's the funny thing, it's not strictly _needed_ , per se, but it does make FSMCollection a lot more usable.

Reply:
Just a comment clarifying that